kakao/src/main/kotlin/com/agoda/kakao/screen/Screen.kt

```diff
@@ -159,8 +200,8 @@ open class Screen<out T: Screen<T>>: ScreenActions {
                     .apply { this(function) }
         }
 
-        internal val vis: Stack<Interceptor<ViewInteraction, ViewAssertion, ViewAction>?> = Stack()
-        internal val dis: Stack<Interceptor<DataInteraction, ViewAssertion, ViewAction>?> = Stack()
-        internal val wis: Stack<Interceptor<Web.WebInteraction<*>, WebAssertion<*>, Atom<*>>?> = Stack()
+        internal val viewInteractionStack: Stack<Interceptor<ViewInteraction, ViewAssertion, ViewAction>?> = Stack()
```

Comment:
I think `viewInterceptorStack` better represents the purpose for this collection.
The same applies to the other stacks